### DISCUSSION TITLE
Problem with run command with specified suite or tests with one runner

### DISCUSSION THREAD

**Discussion by adek23** [Codeception/Codeception#6173] [Ideas]
I followed information on this page to have seperate tests for each bundle: http://codeception.com/docs/08-Customization

Unfortunately **run** command doesn't support this configuration, because
command **run unit** still runs all tests in all suites.
I have to specify configuration in bundle to get this to work.

**Comment by haclong** [comment]
I have the same issue

**Comment by DavertMik** [comment]
What was the expected behavior? 
Run only unit suites from all apps?

**Comment by haclong** [comment]
Hello

I am working with ZF2 and i am bootstraping one codeception test suites per ZF2 modules.
Then i add the path of all my codeception.yml files into the main codception.yml file so, when running the main config file, all suites are run too.

I have built the first test suites successfully but then, when i add a second module to my ZF2 application (therefore a second tests suites), both tests suites are failing. I have do it all over again, (deleting all files created by/for codeception and rebuild them one by one, testing each time i'm adding any codeception helpers and/or modules but now, so far, it is working fine... but i havent add any tests files again yet...)

So i have two scenarii :
A : build first ZF2 module, bootstrap codeception for that module, add codeception modules for unit and acceptance testing (ZF2 module, etc...)
build second ZF2 module, bootstrap codeception for that second module, add codeception modules for testing

-> this scenario will be most likely the one i should logically use while building my full application but it is the one failing me.

B : build all ZF2 modules known so far, bootstrap one codeception per modules, add codeception modules for each codeception test suites at once

-> this scenario is the one working last time i've tried. This is not acceptable since i don't feel i can expand the application that way since i can't afford to delete all tests suites per module and rebuild them one by one each time i code a new module.

Thank you

Date: Sun, 1 Feb 2015 19:27:20 -0800

**Comment by AhmedSamy** [comment]
I have the same problem, even with suite full path, still runs all

**Comment by bizmate** [comment]
I have a similar problem, i can open a separate issue if necessary. In short adding a root codeception.yml that includes other paths does not work. I am using 2.0.16

``` yml
namespace: frontend
include:
  - src/Business/Ecommerce/AuthenticationBundle
  - src/Business/ReservationBundle
paths:
  log: app/logs/codeception
settings:
  colors: false
```

and all i get is 

``` bash
[vagrant@localhost symfony]$ bin/codecept run
Codeception PHP Testing Framework v2.0.16
Powered by PHPUnit 4.7.7 by Sebastian Bergmann and contributors.



  [Codeception\Exception\Configuration]
  Configuration file could not be found.
  Run `bootstrap` to initialize Codeception.



run [-c|--config CONFIG] [--report] [--html [HTML]] [--xml [XML]] [--tap [TAP]]
[--json [JSON]] [--colors] [--no-colors] [--silent] [--steps] [-d|--debug] [--co
verage [COVERAGE]] [--coverage-html [COVERAGE-HTML]] [--coverage-xml [COVERAGE-X
ML]] [--coverage-text [COVERAGE-TEXT]] [--no-exit] [-g|--group GROUP] [-s|--skip
 SKIP] [-sg|--skip-group SKIP-GROUP] [--env ENV] [-f|--fail-fast] [--no-rebuild]
 [--] [<suite>] [<test>]


[vagrant@localhost symfony]$
```

and running bootstrap command

``` shell
[vagrant@localhost symfony]$ bin/codecept  bootstrap

Project is already initialized in '.'
```

**Comment by dynasource** [comment]
are the codeception.ymls located in?:
- src/Business/Ecommerce/AuthenticationBundle/codeception.yml
- src/Business/ReservationBundle/codeception.yml

are the suites initialized inside both central & local?

**Comment by roosht3** [comment]
@DavertMik Any update on this ?
